.eslintrc.json

```diff
@@ -20,6 +20,8 @@
     "no-param-reassign": "off",
     "class-methods-use-this": "off",
     "no-nested-ternary": "off",
-    "no-use-before-define": "off"
+    "no-use-before-define": "off",
+    "react/jsx-props-no-spreading": "off",
+    "no-restricted-syntax": "off"
```

Comment:
Je l'ai désactivé à cause de ça : 
```
for (const applicant of applicants.selectedApplicants) {
      // Using await here to avoid sending too many requests at the same time
      // eslint-disable-next-line no-await-in-loop
      await applicant.inviteBy(format, isDepartmentLevel);
}
```
La règle empêche de faire des boucles simple et veut privilégier un `.forEach` ou un map/reduce
Sauf que pour ce cas précis on voulait volontairement une boucle synchrone. 

Tu préfères que je comment cette règle uniquement dans le fichier en question ? ça permettra d'expliquer plus clairement je pense

Reply:
Oui ça me parait mieux merci 🙏